openmdao/visualization/n2_viewer/gen/NodeInfo.js

```diff
@@ -336,7 +336,8 @@ class NodeConnectionInfo extends NodeInfo {
             newRow.append('td').html(conn.tgt);
         }
 
-        let title = 'Connections';
+        let title = (color == "black") ? "Jacobian" : "Connections";
```

Comment:
Right now this is hard coded to detect if the cell under the cursor is black. Ideally this would use `Style.colors.declaredPartial`, but I haven't been able to get that to work.